### ISSUE TITLE
Screen freezing issue

### ISSUE THREAD

**Issue by Bydml04** [olegos2/mobox#174]
/
[mobox_log.txt](https://github.com/olegos2/mobox/files/14141879/mobox_log.txt)

https://github.com/olegos2/mobox/assets/62351675/775092e0-dc38-422f-9aae-b849e540dc00

Freezes every 5 seconds and I need to open/close notification panel to solve and it happens again. There is log file too is there ant issue about grapich adaptor?

**Comment by toastie000** [comment]
Having the same exact issue. SD8+ Gen 1, 12gb, Android 14. Have tried reinstalling everything - Mobox, Termux and X11, also tried lowering my devices refresh rate to no avail.

https://github.com/olegos2/mobox/assets/55211530/d840ef5b-8f2e-42f7-a2be-57f911275d67

**Comment by toastie000** [comment]
To update, having my phone plugged specifically into my computer seems to stop the issue. Simply having my phone on charge does nothing to stop it.

**Comment by Bydml04** [comment]
did you solve it by plugging your phone to charge? I didn't understand

**Comment by toastie000** [comment]
No, having it in charge doesn't stop it. Having it plugged into my computer seems to stop it, though.

**Comment by z4h1** [comment]
For me it was resolved in settings
MESA_VK_WSI_PRESENT_MODE=relaxed

**Comment by Bydml04** [comment]
> To update, having my phone plugged specifically into my computer seems to stop the issue. Simply having my phone on charge does nothing to stop it.

how did you change this setting?

**Comment by toastie000** [comment]
> how did you change this setting?

Not sure what you mean, but I've managed to fix it. You need to disable Phantom Process killer. This fixed everything. I can't vouch for the second guide as I used the first one FYI.

Use this tutorial if you have a computer: https://www.youtube.com/watch?v=OZny45wLZL4
Use this one if you don't  : https://www.youtube.com/watch?v=vK1Jx9ydi5c&t=308s

**Comment by Bydml04** [comment]
> > how did you change this setting?
> 
> Not sure what you mean, but I've managed to fix it. You need to disable Phantom Process killer. This fixed everything. I can't vouch for the second guide as I used the first one FYI.
> 
> Use this tutorial if you have a computer: https://www.youtube.com/watch?v=OZny45wLZL4 Use this one if you don't : https://www.youtube.com/watch?v=vK1Jx9ydi5c&t=308s

I tried but didn't worked. Do you use Huawei device?

**Comment by toastie000** [comment]
> I tried but didn't worked. Do you use Huawei device?

I have a Vivo - iQOO Neo 8. Not sure why that hasn't worked, what Android version are you on? I can only recommend trying again to make sure you've done it correctly. Also, I think that setting that z4h1 posted has been removed from Mobox, as I can't find it.

**Comment by Bydml04** [comment]
> > I tried but didn't worked. Do you use Huawei device?
> 
> I have a Vivo - iQOO Neo 8. Not sure why that hasn't worked, what Android version are you on? I can only recommend trying again to make sure you've done it correctly. Also, I think that setting that z4h1 posted has been removed from Mobox, as I can't find it.

I use Huawei Matepad 11.5 and it doesn't supports Google services. The problem maybe because of it

**Comment by Bydml04** [comment]
I was pasting wrong codes. I tried to do again as you said and figured out. Thank you so much for the solution

**Comment by toastie000** [comment]
> I use Huawei Matepad 11.5 and it doesn't supports Google services. The problem maybe because of it

I did some research and your tablet is the issue. It's running HarmonyOS, which uses it's own OS. It doesn't use Android whatsoever, not even the kernel. It uses translation layers to use Android apps. Unfortunately you're out of luck on this one.

**Comment by toastie000** [comment]
> I was pasting wrong codes. I tried to do again as you said and figured out. Thank you so much for the solution

You got it to work?

**Comment by toastie000** [comment]
> I connected to charger because of low battery and it started to freezing again :/ I have one waiting system update. I will wait for charge and try again

The commands are designed for Android devices, so they wont work on your tablet. They're 'adb' / 'android debug bridge' commands. You've got a different OS, so the commands aren't compatible.

**Comment by Bydml04** [comment]
> > I connected to charger because of low battery and it started to freezing again :/ I have one waiting system update. I will wait for charge and try again
> 
> The commands are designed for Android devices, so they wont work on your tablet. They're 'adb' / 'android debug bridge' commands. You've got a different OS, so the commands aren't compatible.

but I can use android debug bridge for screen mirroring. If you now scrcpy I use it gradely

**Comment by Den-Well** [comment]
I just installed everything in my Vivo X Fold 2 (SD 8 Gen 2), but I'm not managing to make it work. I start wine and sometimes an black x appears, other times it manages to load desktop window with drives and start menu button, but then it freezes. Can someone help me, please?

**Comment by colemad** [comment]
I write badly in English. The problem can be solved if you disable automatic control of the application launch in the device settings

**Comment by colemad** [comment]
https://github.com/user-attachments/assets/0e9aa222-bd2d-4216-b20d-fd5d3594d62f

Sorry I had to cut it but I think it will be useful for someone
